### PR TITLE
New metadata

### PR DESCRIPTION
**Pull request by VrbeckaD**
+ published (date)
+ hits

### CODE REVIEW

**Comment by VrbeckaD** [comment]
@janrygl :)

File: hp/metadata_collector.py [hunk 1]
```diff
@@ -96,12 +96,20 @@ def get_text(text_path):
             # vyhledá počet slov
             wordcount = re.search(r'<dd class="words">((?:\d*,?)+)</dd>', part).groups(1)[0].replace(",", "")
             subresult.append(wordcount)
+            # vyhledá datum publikování
+            published= re.search (r'<dd class="published">(\d\d\d\d-\d\d-\d\d)</dd>', part).group(1)[0]
+            subresult.append(published)
+            # vyhledá počet kliknutí na povídku
+            hits= re.search(r'<dd class="hits">\d+</dd>', part).group(1)
```
Comment (janrygl) [inline hp/metadata_collector.py]: missing brackets `(\d+<)`

File: hp/metadata_collector.py [hunk 2]
```diff
@@ -96,12 +96,20 @@ def get_text(text_path):
             # vyhledá počet slov
             wordcount = re.search(r'<dd class="words">((?:\d*,?)+)</dd>', part).groups(1)[0].replace(",", "")
             subresult.append(wordcount)
+            # vyhledá datum publikování
+            published= re.search (r'<dd class="published">(\d\d\d\d-\d\d-\d\d)</dd>', part).group(1)[0]
```
Comment (janrygl) [inline hp/metadata_collector.py]: are you sure you need `[]` in the end? I think it will return first digit of year only

**Comment by VrbeckaD** [comment]
new metadata subresult freeform tags
TODO: run and check functioning

**Pull request merged**